site/src/tokens/colors.md

```diff
@@ -21,12 +21,17 @@ Atlas current supports three themes out of the box.
 	<p><code>.background-color-body-medium</code></p>
 </div>
 
-<div class="padding-xl background-color-alternate color-text-overlay-invert">
+<div class="padding-xl background-color-body-learn">
+	<p class="font-weight-bold font-size-h4 margin-bottom-xs">body-learn</p>
+	<p><code>.background-color-body-medium</code></p>
```

Comment:
Class here should be updated too after `accent` change

Reply:
Updated